packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -160,6 +169,16 @@ export default compose( [
 		const icon = blockType.icon;
 		const getAccessibilityLabelExtra = blockType.__experimentalGetAccessibilityLabel;
 
+		const rootBlockId = getBlockHierarchyRootClientId( clientId );
+		const rootBlock = getBlock( rootBlockId );
+		const parentBlockId = getBlock( clientId ).parent;
+		const parentBlock = getBlock( parentBlockId );
+		const isGroupRoot = rootBlock.name === 'core/group';
```

Comment:
It should be enough to just check for inner blocks, no need to check if it is a group block

Reply:
Agree - `const hasRootInnerBlocks = rootBlock.innerBlocks.length !== 0;` should be enough.